tasks/Windows.yml

```diff
@@ -1,6 +1,11 @@
 ---
+- name: Ensure Chocolatey is installed
+  win_chocolatey:
+    name: chocolatey
+    state: present
+
 - name: Install git
   win_chocolatey:
```

Comment:
`chocolatey.chocolatey.win_chocolatey` to be consistent?

Reply:
The linter only fails for Ansible built-in packages. Maybe you're right, I'll do some reading.